wordpress.org/public_html/wp-content/mu-plugins/pub/wporg-robots.php

```diff
@@ -18,7 +18,10 @@ function wporg_robots_txt( $robots ) {
 		           "Allow: /locale/$\n" .
 		           "Allow: /locale/*/glossary/$\n" .
 		           "Allow: /locale/*/stats/plugins/$\n" .
-		           "Allow: /locale/*/stats/themes/$\n";
+		           "Allow: /locale/*/stats/themes/$\n" .
+		           "Allow: /events/rss\n" .
+		           "Allow: /events/feed\n" .
+		           "Allow: /events/*/*\n";
```

Comment:
Would this also work? Since we'll want to allow everything anyway:
```suggestion
		           "Allow: /events/\n";
```

Reply:
The idea with these 3 lines is to enable the RSS feed and the events, but keeping disabled the other URLs, such as edit, delete,...

Reply:
Since Edit/Delete shouldn't be accessible to robots (which are logged out) and shouldn't be returning `200` responses, it's probably safe to just allow `/events/`